src/InvoiceGenerator.BusinessLogic/ClientService.cs

```diff
@@ -31,7 +31,7 @@ public List<ClientViewModel> GetClients()
         public List<ClientNameViewModel> GetClientNames()
         {
 
-            var clientNames = _invoiceDbContext.Clients.ToList();
+            var clientNames = _clientRepository.GetAll();
 
             var nameModels = new List<ClientNameViewModel>();
```

Comment:
:seedling: we could also make a mapper to the specific viewmodel here, too.
No need to worry about it right now, though.